documentation: Add delete a user group instruction. (#22638)
zulip/zulip: templates/zerver/help/user-groups.md

1. Add or remove users (including yourself). Click outside the box
   to save.  Zulip will notify everyone who is added or removed.

{end_tabs}

### Delete a user group

{start_tabs}

{settings_tab|user-groups-admin}

1. Find the group.

> 1. Click the trash icon (<i class="fa fa-trash-o"></i>) in the lower right corner of the group section.

Reviewer: A few corrections are needed to make it consistent with our style guidelines and other docs:
```suggestion
1. Click on the trash (<i class="fa fa-trash-o"></i>) icon in the top right corner of the user group.
```

> 1. Click the Confirm button.

Reviewer: To make it consistent with other docs, let's make this:
```suggestion
1. Approve by clicking **Confirm**."
```

!!! warn ""
    **Note**: If you remove yourself from a user group, you

>     may no longer have permission to modify the user group.

Reviewer: It looks like this warning was accidentally removed from the "Modify a user group" section and it should be added back to it.

>     **Note**: Once you deleted a group, this action cannot be undone.

Reviewer: Let's make this:
```suggestion
    **Note**: This action cannot be undone by anyone in the organization.
```
